x-pack/metricbeat/module/azure/storage/integration_tests/config/main.go

```diff
@@ -0,0 +1,41 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package main
+
+import (
+	"github.com/pulumi/pulumi-azure/sdk/v3/go/azure/core"
+	"github.com/pulumi/pulumi-azure/sdk/v3/go/azure/storage"
+	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
+)
+
+const (
+	resourceGroupName = "observability-beats-test-storage-account"
+	location          = "WestEurope"
+	storageAccount    = "storageobsaccount"
+)
+
+func main() {
```

Comment:
Can this scenario be executed also with `pulumi up` or similar? This way it could also be used for manual testing or development.

Reply:
yes, this is also an option.